Source/View Controller/ScheduleTableViewController.swift

```diff
@@ -40,6 +39,18 @@ internal class ScheduleTableViewController: NavigationBarViewController, Navigat
         self.viewModel.fetchScheduleData()
     }
     
+    // Map button press event
+    @objc func buttonClicked(_ button: UIButton) {
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let nextViewController = storyboard.instantiateViewController(withIdentifier: "MapViewController")
+        
+        // Remove button
+        button.removeFromSuperview()
+        
+        // Navigate to map view controller
+        navigationController?.pushViewController(nextViewController, animated: true)
+    }
+    
```

Comment:
Is this being used for anything?

Reply:
no

Reply:
I removed this code. If you were curious it was for the floating button that I previously implemented for accessing the map view.